package.json

```diff
@@ -9,8 +9,7 @@
     "start": "npm run dev",
     "test": "gulp lint"
   },
-  "main": "ng2-select.js",
-  "typings": "ng2-select.d.ts",
+  "main": "ng2-select.ts",
```

Comment:
@Booyanach please revert this changes, because in next version only
.js and .d.ts file will be serverd